pkg/config/config.go

```diff
@@ -193,6 +197,14 @@ func (c *Config) incorporateUserSettings(u *Config) {
 	if len(u.Ingress.AdmissionPolicy.NamespaceOwnership) != 0 {
 		c.Ingress.AdmissionPolicy.NamespaceOwnership = u.Ingress.AdmissionPolicy.NamespaceOwnership
 	}
+
+	if u.Ingress.Ports.Http != 0 {
+		c.Ingress.Ports.Http = u.Ingress.Ports.Http
+	}
+
+	if u.Ingress.Ports.Https != 0 {
+		c.Ingress.Ports.Https = u.Ingress.Ports.Https
+	}
```

Comment:
There are some unit tests for the config package. Those should be updated as part of this work.